test/e2e/data/e2e_conf.yaml

```diff
@@ -115,7 +115,11 @@ providers:
 variables:
   # used to ensure we deploy to the correct management cluster
   KUBE_CONTEXT: "kind-capo-e2e"
-  KUBERNETES_VERSION: "v1.25.0"
+  KUBERNETES_VERSION: "v1.25.3"
```

Comment:
latest is 1.25.5
https://kubernetes.io/releases/
curious we didn't use it ?due to consistent to existing test version?

Reply:
Good point! I just copied from CAPI. I'll see if I can get it working with v.25.5